src/components/molecule/PieChartView.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { VictoryPie, VictoryContainer } from 'victory'
+
+interface Idata {
+    angle?: number,
+    radius?: number,
+    label?: string,
+    subLabel?: string,
+    fontSize?: number,
+    style?: object
+    quarter?: string,
+    earnings?: number,
+    x?: number,
+    y?: number
+}
+
+export const PieChartView = () => {
+    const data: Idata[] = [
+        { quarter: 'Car', earnings: 13000 },
+        { quarter: 'Pedestrian', earnings: 16500 },
+        { quarter: 'Cyclist', earnings: 14250 },
+        { quarter: 'Public Transport', earnings: 19000 },
+        { quarter: 'Train', earnings: 19000 },
+        { quarter: 'Light Train', earnings: 19000 }
+    ]
+    return (
+        <React.Fragment>
+            <VictoryPie data={data} x='quarter' y='earnings'
+                style={{
```

Comment:
I think moving the style (and any other larger construct) to a constant at the top of the file (`const PIE_CHART_VIEW_STYLE = { ... }`) will be easier to read.